### PR TITLE
Ukrainian translation update

### PR DESCRIPTION
**Pull request by tsunamistate**
Blackouts suck, but now I got enough time to look at new and updated strings
@SourceCodeDeleted your review will be appreciated, as always

### CODE REVIEW

File: Translations/uk.po [hunk 1]
```diff
@@ -1214,222 +1311,388 @@ msgstr ""
...
 msgstr "Гаряча клавіша для чар або таланту."
 
-#: Source/diablo.cpp:1481
+#: Source/diablo.cpp:1611 Source/diablo.cpp:1984
+msgid "Use health potion"
+msgstr "Використати зілля Зцілення"
+
+#: Source/diablo.cpp:1612 Source/diablo.cpp:1985
+msgid "Use health potions from belt."
+msgstr "Використати зілля Зцілення з пояса."
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1356]: This is fine - Does it fit in the game text box?

File: Translations/uk.po [hunk 2]
```diff
@@ -1214,222 +1311,388 @@ msgstr ""
...
+msgid "Cast the active spell."
+msgstr "Накласти активне чарування."
+
+#: Source/diablo.cpp:1920
+msgid "Cancel action"
+msgstr "Відміна"
+
+#: Source/diablo.cpp:1921
+msgid "Close menus."
+msgstr "Закриває меню."
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1575]: Закрити меню

File: Translations/uk.po [hunk 3]
```diff
@@ -1214,222 +1311,388 @@ msgstr ""
...
+msgid "Hold to prevent the player from moving."
+msgstr "Утримуйте, щоб не дати гравцеві рухатися."
+
+#: Source/diablo.cpp:1976
+msgid "Toggle stand ground"
+msgstr "Перемкнути стояння на місці"
+
+#: Source/diablo.cpp:1977
+msgid "Toggle whether the player moves."
+msgstr "Перемикає, чи рухається гравець."
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1623]: Перемкнути чи рухається гравець. -- I am not sure on this one. I cannot think of a better way to say this.

**Comment by SourceCodeDeleted** [comment]
sorry for my delay. 

for the most part it is ok. I added some comments .

File: Translations/uk.po (hunk 1 above)
Comment (tsunamistate) [reply Translations/uk.po:1356]: It does

File: Translations/uk.po (hunk 3 above)
Comment (tsunamistate) [reply Translations/uk.po:1623]: Same here: it's a description of what menu option does, and I think this case fits here
Comment if it doesn't

File: Translations/uk.po (hunk 2 above)
Comment (tsunamistate) [reply Translations/uk.po:1575]: It's a string that describes what menu option does, IMHO this case fits

File: Translations/uk.po (hunk 3 above)
Comment (tsunamistate) [reply Translations/uk.po]: I changed TL to have better description
Перемикає, чи може гравець рухатись.

File: Translations/uk.po (hunk 2 above)
Comment (SourceCodeDeleted) [reply Translations/uk.po:1575]: ah ok - then it is indeed ok )

**Comment by SourceCodeDeleted** [comment]
ok - I think it is good to add )

**Comment by AJenbo** [comment]
Yeah it's still limited by missing LUA support in general

**Pull request merged**